invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/oaipmh/search/SearchResultItem.js

```diff
@@ -42,44 +48,46 @@ class SearchResultItemComponent extends Component {
     const resourceHasActions =
       displayEdit || displayDelete || !isEmpty(actions);
     return (
-      <Table.Row>
-        {columns.map(([property, { text, order }], index) => {
-          return (
-            <Table.Cell key={`${text}-${order}`} data-label={text}>
-              {index === 0 && (
-                <a
-                  href={AdminUIRoutes.detailsView(
-                    listUIEndpoint,
-                    result,
-                    idKeyPath
-                  )}
-                >
-                  {_get(result, property)}
-                </a>
-              )}
-              {index !== 0 && _get(result, property)}
+      <OverridableContext.Provider value={overridenComponents}>
+        <Table.Row>
+          {columns.map(([property, { text, order }], index) => {
+            return (
+              <Table.Cell key={`${text}-${order}`} data-label={text}>
+                {index === 0 && (
```

Comment:
why not having a ternary operator here?

e.g.

```javascript
{
  index === 0
   ? (<a>...</a>)
   : (_get(result,property)
}
```

Reply:
Yes, you are right, I just copy-pasted from invenio-administration to override it, just fixed it, thanks!